generators/php/sdk/src/endpoint/request/WrappedEndpointRequestGenerator.ts

```diff
@@ -93,7 +96,47 @@ export class WrappedEndpointRequestGenerator extends FileGenerator<
                     clazz.addTrait(this.context.phpTypeMapper.convertToTraitClassReference(declaredTypeName));
                 }
             },
-            fileUpload: () => undefined,
+            fileUpload: (fileUpload) => {
+                for (const property of fileUpload.properties) {
+                    property._visit({
+                        file: (fileProperty) => {
+                            fileProperty._visit({
+                                file: (file) => {
+                                    let type = php.Type.reference(this.context.getFileClassReference());
+                                    if (file.isOptional) {
+                                        type = php.Type.optional(type);
+                                    }
+                                    clazz.addField(
+                                        php.field({
+                                            name: this.context.getPropertyName(file.key.name),
+                                            type,
+                                            access: "public"
+                                        })
+                                    );
```

Comment:
nit: We can handle this with a ternary in-line (the same might apply elsewhere):

```suggestion
                                    const type = php.Type.reference(this.context.getFileClassReference());
                                    clazz.addField(
                                        php.field({
                                            name: this.context.getPropertyName(file.key.name),
                                            type: file.isOptional ? php.Type.optional(type) : type,
                                            access: "public"
                                        })
                                    );
```

Reply:
done